docs/reclaimspace.md

```diff
@@ -63,8 +63,38 @@ spec:
 
 ## Annotating PerstentVolumeClaims
 
-An extension to the controller to automatically create `ReclaimSpaceCronJob`
-based on annotation on PersistentVolumeClaim is being planned.
+`ReclaimSpaceCronJob` CR can also be automatically created by adding 
+`reclaimspace.csiaddons.openshift.io/schedule: "@midnight"` annotation
+to PersistentVolumeClaim object.
+
+```
+$ kubectl get pvc pvc-1
```

Comment:
use meaningful name instead of pvc-1, pvc-sample and pvc-1 names are not matching